twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -387,16 +387,11 @@ def _scrooge_scala_library_impl(ctx):
     else:
         all_java = aspect_info.java_info
 
-    # TODO: Remove `scala` field once JavaInfo supports multiple
-    # output jars. https://github.com/bazelbuild/rules_scala/issues/564
```

Comment:
really, for the TODO we just need to delete this? can we have any test that the output jars are being plumbed through correctly when we have multiple jars?